inc/options/class-wpseo-options.php

```diff
@@ -171,25 +171,41 @@ public static function get_option_names() {
 	 * @return  array  Array combining the values of (nearly) all the options
 	 */
 	public static function get_all() {
-		$all_options  = array();
-		$option_names = self::get_option_names();
+		return self::get( self::get_option_names() );
+	}
+
+	/**
+	 * Retrieve one or more options for the SEO plugin.
+	 *
+	 * @param array|string $option_names The option name or an array of option names of the options you want to get.
+	 *
+	 * @static
+	 * @return  array  Array combining the values of the requested options
+	 */
+	public static function get( $option_names ) {
```

Comment:
It would be nice that you can pass the needed options as arguments. In that case you can get them by `func_get_args()` and that always will be an array.